pipeline/vars/common.groovy

```diff
@@ -237,4 +266,68 @@ def sendUMBMessage(def msgType){
     return sendResult
 }
 
+def fetchTier1Compose(){
+    /*
+       Fetch the compose to be tested for tier1, is there is one
+    */
+    def defaultFileDir = "/ceph/cephci-jenkins/latest-rhceph-container-info"
+    def tier0Json = "${defaultFileDir}/RHCEPH-${env.rhcephVersion}-tier0.json"
+    def tier1Json = "${defaultFileDir}/RHCEPH-${env.rhcephVersion}-tier1.json"
+    def tier1FileExists = sh(returnStdout: true, script: """if [ -f '${tier1Json}' ] ; then echo 'yes' ; else echo 'no' ; fi""").trim()
+    println "tierl 1 file status"
```

Comment:
If the file does not exists, we could return the values retrieved from `latest-{env.rhcephVersion}`

Reply:
This is for tier-1 file check, tier0 file is assumed to be existing.